src/iexamine.cpp

```diff
@@ -2233,6 +2233,26 @@ void iexamine::tree_blackjack(player *p, map *m, const tripoint &examp)
     m->ter_set(examp, t_tree);
 }
 
+void iexamine::tree_birch(player *p, map *m, const tripoint &examp)
```

Comment:
Also, you have to add the function declaration to iexamine.h, just add a line `void iexamine::tree_birch(player *p, map *m, const tripoint &examp);` to it, similar for for the other function (this is the other thing the build bot complains about).